flink-python/pyflink/datastream/utils.py

```diff
@@ -66,7 +66,8 @@ def pickled_bytes_to_python_converter(data, field_type):
         elif field_type == Types.FLOAT():
             return field_type.from_internal_type(ast.literal_eval(data))
         elif isinstance(field_type, BasicArrayTypeInfo) or \
-                isinstance(field_type, PrimitiveArrayTypeInfo):
+                isinstance(field_type, PrimitiveArrayTypeInfo) or \
```

Comment:
change to `isinstance(field_type, (BasicArrayTypeInfo, PrimitiveArrayTypeInfo, ObjectArrayTypeInfo))`